firmware/Core/Inc/telecommands/telecommand_definitions.h

```diff
@@ -4,50 +4,50 @@
 
 #include <stdint.h>
 
-typedef enum {
+typedef enum
+{
     TCMD_TelecommandChannel_DEBUG_UART,
     TCMD_TelecommandChannel_RADIO1
 } TCMD_TelecommandChannel_enum_t;
 
 typedef uint8_t (*TCMD_TCMDEXEC_Function_Ptr)(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
-                                         char *response_output_buf, uint16_t response_output_buf_len);
```

Comment:
Avoid whitespace changes, please.

You should reset this file like:

```bash
git checkout main -- firmware/Core/Inc/telecommands/telecommand_definitions.h
```

You'll also need to re-register your telecommand(s) in the table/array below.